app.test.js

```diff
@@ -57,4 +57,28 @@ describe('get /api',()=>{
     })
 })
 })
-    
\ No newline at end of file
+    
+describe('get /api/articles',()=>{
+    test('should return 200 and an array of article objects with the correct properties',()=>{
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({body})=>{
+
+            expect(typeof body).toBe('object')
```

Comment:
Checking the type of `body` is slightly redundant here and you can remove it because body has to be a object to destructure from it on line 69